var/spack/repos/builtin/packages/py-torch-cluster/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTorchCluster(PythonPackage):
+    """This package consists of a small extension library of
+    highly optimized graph cluster algorithms for the use in
+    PyTorch."""
+
+    homepage = "https://github.com/rusty1s/pytorch_cluster"
+    url      = "https://github.com/rusty1s/pytorch_cluster/archive/1.5.7.tar.gz"
+
+    version('1.5.7', sha256='71701d2f7f3e458ebe5904c982951349fdb60e6f1654e19c7e102a226e2de72e')
+
+    variant('cuda', default=False)
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
+    extends('py-torch+cuda', when='+cuda')
+    extends('py-torch~cuda', when='~cuda')
```

Comment:
`depends_on`, not `extends`

Reply:
yeah... I was getting an error with `depends_on` regarding it not being able to find a header file. I wasn't able to figure out what env var wasn't happy in the time I allocated to get the working for the user. Switching to `extends` "fixed" the issue and I haven't had any complaints from the user since the install. I'll revisit.

Reply:
`extends` _implies_ `depends_on`, but it also implies more than that (that you can `spack activate py-torch-cluster` and it will be symlinked to the `py-torch` installation prefix). If you figure out what has trouble finding a header file let me know and I can investigate.